content/en/blog/news/2022-sbom-formats-comparison.md

```diff
@@ -0,0 +1,161 @@
+---
+title: "Software Bill Of Materials Formats"
+date: 2022-07-24
+draft: false
+weight: 200
+description: >
+  Previously, we discussed the main concepts of Software Bill Of Materials and their use cases. Now we talk about the different formats and standards of generating them.
+categories: [blog]
+keywords: [Community, GSoC, 2022, supply chain security, BOM, SBOM]
+slug: "sbom-formats"
+aliases: []
+author: Osama Magdy
+---
+
+## Prerequisite ##
+If you don't understand what is Software Bill of Materials (SBOM), please read this [blog post](/blog/2022/07/24/intro-to-sbom) first.
+
+## Different SBOM formats comparison
+The National Telecommunications and Information Administration (NTIA) in the U.S. defined [minimum requirements for SBOM formats](https://www.ntia.doc.gov/report/2021/minimum-elements-software-bill-materials-sbom):
+
+* Identifying the supplier of the software component.
+* Identifying the details about the version of the component.
+* Including unique identifiers for the component like cryptographic hash functions.
+* Including the relationships between all dependencies inside the component.
+* Including a timestamp of when and by whom the SBOM report was created or last modified
+
+ In this section, we discuss different kinds and formats for SBOM standards and make a brief comparison between them. Three commonly used standards achieved the NTIA minimum requirements for SBOM generation and each one results in a different final SBOM document.
+
+### 1 - The Software Package Data Exchange (SPDX)
+
+***History:***
+
+ The first one is an open-source machine-readable format adopted by the Linux Foundation as an industry standard. The specifications are implemented as a file format that identifies the software components within a larger piece of computer software and fulfilling the requirements of NTIA. The SPDX project started in 2010 and was initially dedicated to solving the issues around open source licensing compliance. It evolved over the years to adhere supply chain security challenges and has seen extensive uptake by companies and projects in the software industry. Companies like Hitachi, Fujitsu, and Toshiba contributed to furthering the standard in the [SPDX v2.2.2 specification release](https://spdx.github.io/spdx-spec/).
```

Comment:
`SPDX is an open-source machine-readable format adopted by the Linux Foundation as an industry standard`

Reply:
Done